### DISCUSSION TITLE
What the hell is "Orbis"?

### DISCUSSION THREAD

**Discussion by JimmyCushnie** [mono/mono#20786] [Q&A] [answered]
Hello,

All over the Mono codebase I see a preprocessor symbol called "ORBIS", and many of the .csproj files contain the line `<PropertyGroup Condition=" '$(Platform)' == 'orbis' ">`. But despite fifteen minutes of searching the web and searching this repo, I cannot figure out what the hell Orbis is. Would highly appreciate some enlightenment, as without knowing when this preprocessor symbol is defined, I can't know when code within it is supposed to run.

Edit: Actually, a much better question would be: is there somewhere I can view a list of all the preprocessor symbols and what they mean? I can't find anything like that in the docs.

**Comment by akoeplinger** [comment, marked as answer]
Orbis is the name for the PlayStation4 OS: https://en.wikipedia.org/wiki/PlayStation_4_system_software

You shouldn't really hit it since it's a closed platform.

**Reply by JimmyCushnie** [reply]
Thank you very much!
